k8s/namerequest-ui-pipeline.param

```diff
@@ -5,10 +5,8 @@
 # Action: build
 # Template File: templates/namerequest-ui-pipeline.json
 #=========================================================
-NAME="namerequest-ui"
-PIPELINE_PURPOSE="build"
+NAME="namerequest"
+TAG="dev"
```

Comment:
When this project is complete, can you give us a quick review on the flow to get code/build from GH to Dev then Test then Prod? I'm wondering about the new pipelines and whether we have to start anything manually in OCP4, etc.